src/atrament.js

```diff
@@ -206,6 +207,7 @@ module.exports = class Atrament extends AtramentEventTarget {
       this.dispatchEvent('strokerecorded', { stroke });
     }
     this.strokeMemory = [];
+    delete (this.strokeTimestamp);
```

Comment:
I'd say the `delete` here isn't needed, as we can just hold on to the last value and overwrite it when the next stroke begins - or am I missing an edge case?

Reply:
Sure, the `delete` isn't needed! I added it for some consistency: if no stroke has started, no `strokeTimestamp` is defined. I can remove this line if you prefer!

Reply:
that actually sounds reasonable, thank you! will merge this now and release the new version